src/renderer/mod.rs

```diff
@@ -60,7 +60,16 @@ impl Renderer {
         }
     }
 
+    #[cfg(not(feature = "testing-colors"))]
     /// Default terminal styling
+    ///
+    /// # Note
+    /// Some colors are OS dependant, meaning that the colors will look
+    /// different on different operating systems, making testing colored output
+    /// difficult.
+    /// If you are testing the colored output, you can use the
+    /// [`testing-colors` feature](crate#features) to enable colors that are OS
+    /// independent.
     pub const fn styled() -> Self {
         const BRIGHT_BLUE: Style = if cfg!(windows) {
```

Comment:
Should we try to single-source these?  If nothing else, we could have an internal `const fn` that has a `windows: bool` parameter.